nextroute/solver_parallel.go

```diff
@@ -9,8 +10,8 @@ import (
 
 // ParallelSolveOptions are the options for the parallel solver.
 type ParallelSolveOptions struct {
-	MaximumDuration      time.Duration `json:"maximum_duration" usage:"maximum duration of solver in seconds" default:"30s"`
-	MaximumParallelRuns  int           `json:"maximum_parallel_runs" usage:"maximum number of parallel runs, -1 implies using all available resources" default:"100"`
+	MaximumDuration      time.Duration `json:"maximum_duration" usage:"maximum duration of the solver" default:"30s"`
+	MaximumParallelRuns  int           `json:"maximum_parallel_runs" usage:"maximum number of parallel runs, -1 results in using all available resources" default:"100"`
 	StartSolutions       int           `json:"start_solutions" usage:"number of solutions to start with; one solution generated with sweep algorithm, the rest generated randomly" default:"0"`
```

Comment:
Let's 0 be actually no start solutions and make -1 be the number of CPU's (which I believe 0 implies today)